util/test/account_testutil.go

```diff
@@ -71,8 +71,8 @@ func DecodeAddressOrPanic(addr string) basics.Address {
 	return result
 }
 
-// MakeAssetConfigOrPanic is a helper to ensure test asset config are initialized.
-func MakeAssetConfigOrPanic(round, configid, assetid, total, decimals uint64, defaultFrozen bool, unitName, assetName, url string, addr basics.Address) (*transactions.SignedTxnWithAD, *idb.TxnRow) {
+// MakeCreateAssetTxn is a helper to ensure test asset config are initialized.
+func MakeCreateAssetTxn(round, configid, assetid, total, decimals uint64, defaultFrozen bool, unitName, assetName, url string, addr basics.Address) (*transactions.SignedTxnWithAD, *idb.TxnRow) {
```

Comment:
This is AssetConfig not create. Create is a special case of asset config when the asset ID is 0.

Reply:
thanks